src/specs/components/file/demo/data--without-translation.js

```diff
@@ -4,6 +4,7 @@ const exampleLink = `${publicUrl}/example`;
 module.exports = {
   title: 'State of the Union 2018 brochure',
   aria_label: 'Download file State of the Union 2018 brochure',
+  detail_meta: ['Resource type', 'Publication date'],
```

Comment:
before we recieve feedback, I'd suggest to put only the date here, as it is on figma

Reply:
also in figma you have the meta + date, this is also demoing the right "spacing" between the items, i think we can leave it like this, also because for the mess we make in the story this gets currently replaced everytime with the same value coming from another data file.